issue-creator/cronjob.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: extensions/v1beta1
+kind: CronJob
```

Comment:
Thats it. It needs to be able to list the collaborators and valid labels on the repo for validation though. I'm going to try to make it a prow job and see if everything works with less permissions.